Automattic/sensei
# Disable Divi Theme Builder when Learning Mode enabled.

Fixes #6414

### Changes proposed in this Pull Request
* Disable Divi Theme Builder layout when learning mode is enabled.

### Testing instructions
* Add a global header under Divi > Theme Builder in the "Default Website Template".
* Enable Learning Mode.
* Go to a Lesson and edit using Divi Builder.
* You should be able to edit the lesson correctly and the global header **must not appear** – see previous error in #6414.
* When Learning Mode is disabled the global header **must appear** and the editor will work correctly too.

### Screenshot / Video
![image](https://user-images.githubusercontent.com/799065/215817787-21c775ea-309a-48d5-8c4d-3b6a20a54a5d.png)
(Divi builder loads correctly when editing a Lesson).
